README.md

```diff
@@ -1,13 +1,11 @@
 <a href="http://tarantool.org">
 	<img src="https://avatars2.githubusercontent.com/u/2344919?v=2&s=250" align="right">
 </a>
-<!--a href="https://travis-ci.org/tarantool/go-tarantool">
-	<img src="https://travis-ci.org/tarantool/go-tarantool.png?branch=master" align="right">
-</a-->
 
+[![Go Reference](https://pkg.go.dev/badge/github.com/tarantool/go-tarantool.svg)](https://pkg.go.dev/github.com/tarantool/go-tarantool)
```

Comment:
I don't get how this could be useful to a user
![image](https://user-images.githubusercontent.com/20455996/161787611-5c0a27c4-dc33-494d-83af-a9466e1f8063.png)